qcelemental/models/molecule.py

```diff
@@ -91,17 +89,17 @@ class Molecule(BaseModel):
     molecular_multiplicity: int = 1
 
     # Atom data
-    masses: List[float]
-    real: List[bool]
-    atom_labels: Optional[List[str]] = None
-    atomic_numbers: Optional[List[int]] = None
-    mass_numbers: Optional[List[int]] = None
+    masses: Optional[Array[float]] = None
```

Comment:
Small thing that is more a semantic thing. `Optional` and `= None` do the same thing from Pydantic's versions. But I wonder if having both makes it easier to understand from a user perspective. More of a thought than a request for change.

Reply:
Does `= None` make it an optional? I recall trying this and running into issues.

Reply:
Right, so this is required for these cases. If you do `= None` it will assign the value to None during __init__; however, if you have `Optional` it will allow the value to be set to None in the after an init and by passing in `masses=None` to the constructor. This is a delicate issue, but required.